Makefile

```diff
@@ -223,6 +224,12 @@ ifneq ($(IN_DOCKER),)
 endif
 	docker exec -t -i ${DOCKER_NAME} ./manage.py shell
 
+dbshell:
+ifneq ($(IN_DOCKER),)
+	$(warning Command is designed to be run in the host)
+endif
+	docker exec -t -i ${DOCKER_NAME} ./manage.py dbshell
+
```

Comment:
Yeah, I was planning on keeping it as a separate commit. I won't squash it with the rest of it.